src/pages/ListScholarships.jsx

```diff
@@ -44,15 +45,15 @@ function ListScholarships() {
 
   const pruneQueryParam = (index) => {
     delete params[index];
-    history.replace({ search: queryString.stringify(params) });
+    history.replace({ search: queryString.stringify(params, queryOptions) });
   };
 
   /**
    * when dealing with invalid values in an array
    * rather than pruning the entire param, we can prune out the invalid values
    */
   const replaceQueryParam = (index, newVal) => {
```

Comment:
Hmm `index` feels kind of weird. Could you rename these functions' `index` parameter to `field` or `key`?